packages/elastic-apm-synthtrace/src/scripts/examples/01_simple_trace.ts

```diff
@@ -38,8 +39,16 @@ const scenario: Scenario = async (runOptions: RunOptions) => {
             .duration(1000)
             .success()
             .children(
+              instance
+                .span('GET /products', 'sql', 'mysql')
+                .defaults({ 'span.db.statement': 'SELECT * FROM PRODUCTS' })
+                .duration(1000)
+                .success()
+                .destination('elasticsearch')
+                .timestamp(timestamp),
               instance
                 .span('GET apm-*/_search', 'db', 'elasticsearch')
+                .defaults({ 'span.db.statement': JSON.stringify(mockDbStatement) })
```

Comment:
I think the format for ES requests is slightly different. Do you mind having a look at opbeans data and creating a function that formats it in the way the agent does? I think it's something like query parameters url encoded on the first line, then the actual request body on the second line.